tox.ini

```diff
@@ -8,6 +8,7 @@ indexserver =
 deps =
     :preinstall: -rrequirements.txt
     -rcompanion.txt
+    mkl
```

Comment:
@ahnitz It might make sense to put it here, especially is it may not be clear to some users that this might need installing on some systems to access MKL (although adding it to companion.txt may not solve that problem). At the moment though, MKL is handled specially (it's installed specifically systemwide on the linux system ... but as mac have made `/usr` read-only for even the root user in modern OSs the same thing cannot be done there).

Therefore I'd prefer to leave things like this for this patch, and consider centralizing how/where mkl is installed in another patch if needed .... It's not clear to me that mkl installed like this is slower than systemwide installs? If it is we probably don't want to favour this, but if these MKL libraries are just as fast as whatever we're using now then `pip/conda install mkl` is much easier to manage!